core/src/test/resources/OpenApi2QuarkusTest/_expected-full-different-ns/generated-api/src/main/java/io/apicurio/registry/types/RuleType.java

```diff
@@ -0,0 +1,48 @@
+
+package io.apicurio.registry.types;
```

Comment:
Those types are still generated in the `io.apicurio.registry.types` package, I think this is a bug, cc. @EricWittmann

Reply:
Why do you think this is a bug?

Reply:
Because I'm generating the output explicitly specifying the target package:

https://github.com/Apicurio/apicurio-codegen/pull/204/files#diff-5de8594ac5a967e7b235d3cff81c3b1a15298617ca1cfa369bde7be3e16a0531R45